lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -1035,6 +1035,14 @@ pub unsafe extern "C" fn handle_tdp_sd_read_response(
     client_ptr: *mut Client,
     res: CGOSharedDirectoryReadResponse,
 ) -> CGOErrCode {
+    // # Safety
```

Comment:
I feel like there are too many of these comments. Do you really think we need all of them or is the pattern clear now?

Reply:
How about moving it to the module level documentation:

4b8abcf9205a8499c681e5a44429216518a36f83